### PR TITLE
Update pyproject.toml.jinja

### PR DESCRIPTION
**Pull request by drewoldag**
Fixing typo and bug in the template's pyproject.toml file.

### CODE REVIEW

**Comment by drewoldag** [comment]
Any thoughts on this one would be appreciated. Clearly there is a typo in the pyproject.toml file, and I believe that setuptools will ignore the namespace section as a result of that. Locally, when I make the same changes, everything seems to work and the tests pass. 

When this change isn't in place, I _think_ that python is creating an "implicit namespace package". So it looks like everything is working. But I'm concerned that now if we try to fix it, we are un-implicit-namespace-ifying things and things start to break. 

Perhaps thinking clearly about this overnight will help.

**Comment by delucchi-cmu** [comment]
When I try to hydrate a project from this branch locally, I get the same errors as in the github action. I tried playing around with various permutations of the old and new, and /src/rail/ and /rail/, and mostly just got frustrated =P

LMK if sleep fixes it.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,10 +49,13 @@ jobs:
...
       run: |
         cd ../test/${{ matrix.copier_config.foldername }}
+        # initialize git
         git config --global user.email "tstname@example.com"
         git config --global user.name "Test Name"       
         git init --initial-branch=main
-        git add README.md .copier-answers.yml
+        # commit src which seems required so that site-utils contains rail/example_package
+        # and import rail.example_package will function
+        git add src
```
Comment (cchris28) [inline .github/workflows/ci.yml:58]: My best guess is that the version creation code checks for the existence of the subpackage before it creates artifacts "rail/example_package". But I can't explain why I had some success with the initial commit of README.md alone.

**Review by drewoldag — COMMENTED** [review]
Looks good. I cannot approve this PR though, because I created it. @delucchi-cmu if you have a chance, could you give this the green check?

**Review by cchris28 — APPROVED** [review]

**Review by delucchi-cmu — APPROVED** [review]

**Pull request merged**